pyflakes/test/test_api.py

```diff
@@ -55,26 +55,50 @@ def __init__(self, lineno, col_offset=0):
 
 class SysStreamCapturing(object):
 
-    """Replaces sys.stdin, sys.stdout and sys.stderr with StringIO objects."""
+    """
```

Comment:
My understanding is that PEP 257 doesn't recommend either approach for the summary line (same as opening line, or next line).
This code base very obviously uses the latter convention, which is why I have done it.